app/src/main/java/org/listenbrainz/android/ui/screens/brainzplayer/BrainzPlayerBackDropScreen.kt

```diff
@@ -266,6 +265,32 @@ fun PlayerScreen(
                     onValueChanged = brainzPlayerViewModel::onSeeked
                 )
             }
+            Row(verticalAlignment = Alignment.CenterVertically,
+                horizontalArrangement = Arrangement.SpaceBetween,
+                modifier = Modifier
+                    .fillMaxWidth(0.98F)
+                    .padding(start = 10.dp, top = 10.dp, end = 10.dp)) {
+                val song by brainzPlayerViewModel.currentlyPlayingSong.collectAsState()
+                var duration = "00:00"
+
+                Text(
+                    text = "00:00",
```

Comment:
Shouldn't this represent the current duration at which the song is when playing?

Reply:
Yup, missed that. Added that in the recent commit. Have look at it.